apps/ejabberd/src/ejabberd_c2s.erl

```diff
@@ -1722,8 +1724,8 @@ send_and_maybe_buffer_stanza({J1, J2, El}, State, StateName)->
             maybe_enter_resume_session(BufferedStateData#state.stream_mgmt_id, BufferedStateData)
     end.
 
-send_result_to_amp_event(ok) -> delivered;
-send_result_to_amp_event(_) -> delivery_failed.
+result_to_amp_event(ok) -> delivered;
```

Comment:
I beg to differ - it confused me a great deal because I thought it was meant to send something, while all it does is change one atom into another.